warehouse/query-core/src/main/java/datawave/query/jexl/functions/GeoFunctionsDescriptor.java

```diff
@@ -300,8 +300,10 @@ public JexlNode rebuildNode(ShardQueryConfiguration settings, MetadataHelper met
                             
                             // if there are other fields, recreate the geo function node
                             if (!otherFields.isEmpty()) {
-                                JexlNode geoNode = JexlASTHelper.parseJexlQuery(namespace + ":" + name + "(" + getFieldParam(otherFields) + ", '"
-                                                + args.get(1).image + "', '" + args.get(2).image + "')");
+                                // using dereference of the child node since we do not want the JexlASTScript not the JexlASTReference parent nodes
```

Comment:
```suggestion
                                // using dereference of the child node since we do not want the JexlASTScript nor the JexlASTReference parent nodes
```